tests/test_unit/test_connections/connection_fixtures/group_connections_fixture.py

```diff
@@ -17,13 +22,30 @@ async def group_connections(group_connection: MockConnection, session: AsyncSess
     # since group_connection already created a connection, we start from index 1
     for conn_type in connection_types[1:]:
 
-        new_data = {  # TODO: create different dicts
+        new_data = {
             **connection.data,
             "type": conn_type.value,
-            "cluster": "cluster",
-            "bucket": "bucket",
         }
 
+        if conn_type in [ConnectionType.HDFS, ConnectionType.HIVE]:
+            new_data.update(
+                {
+                    "cluster": "cluster",
+                },
+            )
+        elif conn_type == ConnectionType.S3:
+            new_data.update(
+                {
+                    "bucket": "bucket",
+                },
+            )
+        elif conn_type in [ConnectionType.ORACLE, ConnectionType.CLICKHOUSE, ConnectionType.MSSQL]:
```

Comment:
Postgres is missing

Reply:
Actually, first group_connection fixture already created a connection with Postgres, but it still should be checked if we will change this fixture, added

Reply:
Why some DB connections have `database_name` field, and others have `database`?

Reply:
I think `database` is more appropriate, let's use this option

Reply:
A lot of places need to be fixed for name congruence, should it be done in a separate PR?

Reply:
> A lot of places need to be fixed for name congruence, should it be done in a separate PR?

Yep

Reply:
For Oracle fields are `service_name` and `sid`. For other JDBC types they should have the same name, either `database_name` or `database`. IMHO `database_name` is more descriptive.